src/plugins/three/QuantizedMeshPlugin.js

```diff
@@ -150,7 +150,7 @@ export class QuantizedMeshPlugin {
 
 				this.layer = json;
 
-				if ( json.extensions.length > 0 ) {
+				if ( json.extensions?.length > 0 ) {
```

Comment:
Similar to above, lets extract all the fields of the json into local variables and default them. I think it makes it easier to follow what's happening:

```js
const {
  bounds,
  projection = 'EPSG:4326',
  extensions = []
} = json;
```

I don't suppose you know of a specification for QMesh's layer.json anywhere? It seems fairly adhoc.